extensions/blocks/podcast-player/view.js

```diff
@@ -31,32 +39,37 @@ const initializeBlock = function( id ) {
 	// Load data from the embedded JSON.
 	const dataContainer = block.querySelector( 'script[type="application/json"]' );
 	if ( ! dataContainer ) {
+		downgradeBlockToStatic( block );
```

Comment:
It's cool that we do this. Is there no way to leave the Block in default mode first and then enhance it.

So if the initialisation works then run the code which adds the `enhanced` class otherwise leave the `unenhanced` class in place (or `no-js`).

Seems backwards having to downgrade something.

Reply:
See also https://github.com/Automattic/jetpack/pull/15137#pullrequestreview-382125253